cli/templateedit.go

```diff
@@ -58,11 +76,12 @@ func templateEdit() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&name, "name", "", "", "Edit the template name")
-	cmd.Flags().StringVarP(&displayName, "display-name", "", "", "Edit the template display name")
-	cmd.Flags().StringVarP(&description, "description", "", "", "Edit the template description")
-	cmd.Flags().StringVarP(&icon, "icon", "", "", "Edit the template icon path")
-	cmd.Flags().DurationVarP(&defaultTTL, "default-ttl", "", 0, "Edit the template default time before shutdown - workspaces created from this template to this value.")
+	cmd.Flags().StringVarP(&name, "name", "", "", "Edit the template name.")
+	cmd.Flags().StringVarP(&displayName, "display-name", "", "", "Edit the template display name.")
+	cmd.Flags().StringVarP(&description, "description", "", "", "Edit the template description.")
+	cmd.Flags().StringVarP(&icon, "icon", "", "", "Edit the template icon path.")
+	cmd.Flags().DurationVarP(&defaultTTL, "default-ttl", "", 0, "Edit the template default time before shutdown - workspaces created from this template default to this value.")
+	cmd.Flags().DurationVarP(&maxTTL, "max-ttl", "", 0, "Edit the template maximum time before shutdown - workspaces created from this template must shutdown within the given duration after starting. This is an enterprise-only feature.")
```

Comment:
> This is an enterprise-only feature

nit: do we need to mention it? If we decide to open this feature to the OSS audience, it would be misleading. I'm not sure what's the rule of thumb for enterprise features in general.

Reply:
Seems weird to not mention it if it's a licensed feature, otherwise people will try `--max-ttl` and see errors. The frontend does something similar, by showing the box but disabling it and adding a "enterprise" label

Reply:
Is it common for other flags? I might have missed it.

Reply:
I have done it for deployment config flags before but I'm unsure if we've done it for CLI flags.